synapse/tests/test_lib_types.py

```diff
@@ -367,6 +369,38 @@ def test_range(self):
         self.raises(s_exc.BadTypeDef, model.type('range').clone, {'type': None})
         self.raises(s_exc.BadTypeDef, model.type('range').clone, {'type': ('inet:ipv4', {})})  # inet is not loaded yet
 
+    async def test_range_filter(self):
+
+        async with self.getTestCore() as core:
+
+            async with await core.snap() as snap:
+                node = await snap.addNode('inet:ipv4', '3.3.3.3')
```

Comment:
When testing more primordial model features, it is preferred to use the test model.  This tends to reduce test code churn if we have to change any of the regular synapse models.  A list of the current forms can be seen here:

https://github.com/vertexproject/synapse/blob/010/synapse/tests/utils.py#L120

`testint` may be a suitable candidate for your primary property tests; and anything with an integer secondary property would work for your secondary property tests - using `teststr` and setting a `tick` secondary property on the nodes would work for that.

In your test cases below you are testing the filtration of values for things which have a normed system value which is easily represented as a integer.

Reply:
Cool. I'll switch those around.